libs/core/mappers/sorting-params.mapper.ts

```diff
@@ -0,0 +1,25 @@
+import { SortFieldDto, SortingParamsDto } from '../dtos/sorting-params.dto';
+import { Direction, SortField, SortingParams } from '../models/sorting-params';
+
+export namespace SortingParamsMapper {
+
+	/**
+	 * Maps model to dto.
+	 * @param model SortingParams model.
+	 */
+	export function toDto(model: SortingParams): SortingParamsDto {
+		const direction = model.direction === Direction.Descending ? '-' : '';
+		const sortParamsDto = SORT_PARAMS_TO_DTO[model.activeSortField];
+		return ({
+			ordering: direction + sortParamsDto,
+		});
+	}
+}
+
+/** Sort field type transformation object from model to dto. */
+const SORT_PARAMS_TO_DTO = {
```

Comment:
In fact, such objects are also mappers.
```suggestion
const SORT_PARAMS_TO_DTO_MAP = {
```